tests/resources/screenshot-override/override.js

```diff
@@ -18,4 +18,8 @@ $(document).ready(function () {
     } else if ($.Velocity) {
         $.Velocity.mock = true;
     }
+
+    $('.entityTable.system-check tr:contains(Time) td:nth-child(2)').text('Not showing in tests');
```

Comment:
it works for me when I execute it manually but seems to not yet work in screenshot tests